.github/workflows/test.yml

```diff
@@ -13,5 +13,12 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0
-    - uses: actions/setup-dotnet@v1
-    - run: dotnet test -c Release
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+    - name: Dotnet Test
+      run: dotnet test -c Release /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura /p:CoverletOutput=TestResults/ 
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v1
+      if: ${{ matrix.os == 'macos-latest' }}
+      with:
```

Comment:
We have other test project other than `docfx.Test`. `codecov` should be able to automatically grab the coverage files, so would remove this parameter and run `dotnet test` without `/p:CoverletOutput` work?

Reply:
lemme try